### PR TITLE
[MKLDNN] fix padding calculation with dilation

### PR DESCRIPTION
**Pull request by huningxin**
fix https://github.com/intel/webml-polyfill/issues/551

### CODE REVIEW

**Comment by huningxin** [comment]
PR Submission Checking Summary
Baseline Information:
Chromium version: f8ebaa5
Webml-polyfill version: 9b002bf
PR Submission Proposal:
    ➤   Linux-clDNN: OK
    ➤   Linux-MKLDNN: OK
    ➤   Linux-WASM: OK
    ➤   Linux-WebGL: OK
PR Submission Message:
Total Test Cases: 2632
Pass to Fail: 0
Fail to Pass: 9


Feature | TestCase | Baseline | Linux-clDNN
-- | -- | -- | --
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 1 h3 w2 implicit padding same example-3 | Fail | Pass
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 1 h3 w2 implicit padding same example-4 | Fail | Pass
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 3 h3 w2 implicit padding same example-3 | Fail | Pass
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 3 h3 w2 implicit padding same example-4 | Fail | Pass


Feature | TestCase | Baseline | Linux-MKLDNN
-- | -- | -- | --
CTS Supplement Test | CTS Supplement Test-check result for Reshape example | Fail | Pass
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 1 h3 w2 implicit padding same example-3 | Fail | Pass
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 1 h3 w2 implicit padding same example-4 | Fail | Pass
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 3 h3 w2 implicit padding same example-3 | Fail | Pass
CTS Supplement Test | CTS Supplement Test-check result for ATROUS_CONV_2D 3 h3 w2 implicit padding same example-4 | Fail | Pass

**Comment by huningxin** [comment]
@fujunwei , PTAL. Thanks!

File: services/ml/compilation_impl.cc [hunk 1]
```diff
@@ -26,12 +26,14 @@ inline void CalculateExplicitPadding(bool padding_same,
   padding_head = 0;
   padding_tail = 0;
 
+  uint32_t effective_filter_size = (filter_size - 1) * dilation + 1;
```
Comment (fujunwei) [inline services/ml/compilation_impl.cc]: It's better to close usage, so move to l.33.

Comment (huningxin) [reply services/ml/compilation_impl.cc]: thanks! will fix

**Comment by huningxin** [comment]
@fujunwei , addressed your comment. Please take another look. Thanks!

**Comment by fujunwei** [comment]
LGTM, thanks.

**Comment by huningxin** [comment]
Thanks!

**Pull request merged**